### DISCUSSION TITLE
Saran bootcamp online/offline yang free/paid untuk web developer fullstack

### DISCUSSION THREAD

**Discussion by fadhil-khusnul** [deaafrizal/forum#7] [The Lounge]
Permisi abang semuanya, mumpun lagi semangat belajar dan nyari-nyari pengalaman, mohon saran dan info untuk event bootcampnya yang bagus dimana aja dong 🙏

**Comment by teguhkrniawan** [comment]
Kalau kamu cukup punya finance yang berlebih kamu bisa ikut bootcamp hactiv8 atau purwadhika. Menurutku brand ini cukup punya nama di indo, mereka punya service di offline ataupun online, tapi harganya menurutku mahal dan itu sebanding kok dengan apa yang didapatkan. 

Kalau yang free kamu coba pentengin sosial media digitalent dari Kominfo, dalam setahun biasanya akan ada pelatihan berbentuk bootcamp yang bisa kita ikuti apabila memenuhi persyaratannya. Dan persayaratannya gak susah kok, juga ada testnya sih, tapi menurutku testnya 99% lolos deh.
